DESCRIPTION

```diff
@@ -39,4 +39,5 @@ Imports:
     qmongrdata (>= 0.0.0.9000)
 Depends: R (>= 3.5)
 Remotes:  
-    skde-felles/qmongrdata
+    skde-felles/qmongrdata,
+    github::rstudio/shiny
```

Comment:
['Remotes' defaults to github repos by default](https://github.com/r-lib/remotes/blob/master/vignettes/dependencies.Rmd), no need to define it

Reply:
Thanks!